### PR TITLE
'friendlier' default color scheme

### PR DESCRIPTION
**Pull request by laktak**
I replaced the default `magenta` with a more neutral `lightblack` - see #72

### CODE REVIEW

**Comment by secrettriangle** [comment]
Maybe my terminal configuration is different, but `lightblack` appears just like `black` on my screen, so it's impossible to distinguish between the file browser, editor, and header. That being said, I am open to changing the default color scheme considering the fact that `magenta` is too bright with some terminal color schemes.

File: slap.ini [hunk 1]
```diff
@@ -170,8 +170,13 @@ focusPrev[] = "left"
 ;;;;;;;;;;
 ; Styles ;
 ;;;;;;;;;;
+
```
Comment (secrettriangle) [inline slap.ini]: Thanks for this, I will be sure to merge this comment in.

**Comment by laktak** [comment]
Finding a default that works everywhere will probably be difficult. 

Are you planning on adding themes at some point? Allowing the user to select from some predefined themes might work better.

btw, I found the ini format a bit dated, maybe you could consider [Hjson](http://laktak.github.io/hjson/) (shameless plug) or something JSON like?

**Comment by secrettriangle** [comment]
It will indeed be difficult (but necessary) to find a less contrasting default that works everywhere. Unfortunately it seems `lightblack` renders as `black` on GNOME terminal and xterm with default settings on Ubuntu.

Theming will be implemented with #30.

If you want to write `~/.slap/config` in JSON, go ahead! [`rc`](https://github.com/dominictarr/rc) supports both INI and JSON config files.

File: slap.ini (hunk 1 above)
Comment (secrettriangle) [inline slap.ini]: Added: fd9f7cb

**Comment by laktak** [comment]
I tried a few combinations but none of them worked on both OS X, Linux and Windows (is Windows supported? It often crashes without an error) so I'm going to close this.

**Comment by secrettriangle** [comment]
@laktak From my testing, the only Windows terminal I've found that works with slap is PuTTY.

**Comment by taneltm** [comment]
@secrettriangle What about the telnet client on windows? - It's different from cmd.exe
Slap could start a telnet server and open the windows telnet client.

It wouldn't be a clean solution, but it might be an alternative to PuTTY - an alternative to downloading an additional application.

**Comment by secrettriangle** [comment]
@taneltm slap works with `cmd.exe` but not very well. It doesn't support resizing or mouse events (see [chjj/blessed#windows-compatibility](https://github.com/chjj/blessed#windows-compatibility)) and it's buggy. See #109.

**Pull request closed without merging**